inc/itilfollowup.class.php

```diff
@@ -254,6 +254,12 @@ function post_addItem() {
          ]);
       }
 
+      if ($donotif) {
```

Comment:
Maybe it should be placed after the next block. Indeed, `$donotif = false; // Done for ITILObject update (new status)` have to be evaluated first.